### PR TITLE
Albumnetation bumped up

### PR DESCRIPTION
**Pull request by DwijayDS**
# Requirements

1. Update Albumentation backend and ops
2 .Bring new things from Albumentation to FE

* Users
* Developers
* Admins

# Design / Implementation Details

None

# Known Issues / Limitations

None

### Summary by CodeRabbit

- New Feature: Added several new image augmentation operations, enhancing the software's image processing capabilities. These include bounding box safe random cropping, random cropping from borders, advanced blurring, camera defocus, Fourier Domain Adaptation for style transfer, histogram matching, pixel-level domain adaptation, and zoom blur.
- Improvement: Updated existing image augmentation operations with additional parameters for more control and flexibility.
- Improvement: Updated the 'albumentations' dependency to version 1.3.1, ensuring compatibility with the latest image augmentation techniques.
- Test: Added and updated unit tests for the new and updated image augmentation operations, ensuring their correct functionality.
- Chore: Removed the outdated `iaa_crop_and_pad` module, streamlining the codebase.

### CODE REVIEW

File: test/PR_test/unit_test/op/numpyop/univariate/test_blur.py [hunk 1]
```diff
@@ -35,6 +35,20 @@ def test_single_input(self):
...
+                    sigmaX_limit = (0.5,1),
+                    sigmaY_limit = 1,
+                    rotate_limit = 45,
+                    beta_limit = (0.5, 8.0),
+                    noise_limit = (0.5, 1.0))
+        output = blur.forward(data=self.single_input, state={})
+        with self.subTest('Check output type'):
+            self.assertEqual(type(output), list)
+        with self.subTest('Check output image shape'):
+            self.assertEqual(output[0].shape, self.single_output_shape)
```
[the remark this answers is not in the document]
Comment (DwijayDS) [reply test/PR_test/unit_test/op/numpyop/univariate/test_blur.py:50]: How do I compare the blurred image with input image?

[the remark this answers is not in the document]
Comment (DwijayDS) [reply test/PR_test/unit_test/op/numpyop/univariate/test_blur.py:50]: @coderabbitai how do I compare the blurred image with normal option and advanced

File: fastestimator/op/numpyop/univariate/zoom_blur.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        ds_id: What dataset id(s) to execute this Op in. To execute regardless of ds_id, pass None. To execute in all
+            ds_ids except for a particular one, you can pass an argument like "!ds1".
+    """
+    def __init__(self,
+                 inputs: Union[str, Iterable[str]],
+                 outputs: Union[str, Iterable[str]],
+                 mode: Union[None, str, Iterable[str]] = None,
+                 ds_id: Union[None, str, Iterable[str]] = None,
+                 max_factor: Union[float, Tuple[float, float]] = (1,1.31),
+                 step_factor: Union[float, Tuple[float, float]] = (0.01,0.03)
```
[the remark this answers is not in the document]
Comment (DwijayDS) [reply fastestimator/op/numpyop/univariate/zoom_blur.py:49]: @coderabbitai how should I explain them?

[the remark this answers is not in the document]
Comment (DwijayDS) [reply fastestimator/op/numpyop/univariate/zoom_blur.py:49]: how useful is this class?

File: test/PR_test/unit_test/op/numpyop/univariate/test_blur.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (DwijayDS) [reply test/PR_test/unit_test/op/numpyop/univariate/test_blur.py:50]: @coderabbitai should I include any other test case?

File: fastestimator/op/numpyop/multivariate/crop_and_pad.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 The FastEstimator Authors. All Rights Reserved.
+# Copyright 2023 The FastEstimator Authors. All Rights Reserved.
```
Comment (TortoiseHam) [inline fastestimator/op/numpyop/multivariate/crop_and_pad.py:1]: copyright date can stay as its original value

File: fastestimator/op/numpyop/multivariate/crop_and_pad.py [hunk 4]
```diff
@@ -14,14 +14,15 @@
 # ==============================================================================
 from typing import Iterable, List, Optional, Tuple, Union
 
-from albumentations.imgaug.transforms import IAACropAndPad as IAACropAndPadAlb
+# from albumentations.imgaug.transforms import IAACropAndPad as IAACropAndPadAlb
```
Comment (TortoiseHam) [inline fastestimator/op/numpyop/multivariate/crop_and_pad.py:17]:
```suggestion
```

**Review by TortoiseHam — APPROVED** [review]
LG

**Pull request merged**